extensions/wikia/PowerUser/PowerUser.class.php

```diff
@@ -69,6 +71,22 @@ function __construct( \User $oUser ) {
 	}
 
 	/**
+	 * @var UserPermissions
+	 */
+	private $permissionsService;
```

Comment:
Ok, this code is deleted now and move to the trait, so I'm assuming the issue is resolved